### ISSUE TITLE
failure to install on Macbook M3

### ISSUE THREAD

**Issue by MLeiY** [BradyAJohnston/MolecularNodes#350]
**Describe the bug**

**To Reproduce**

1, Install [miniconda](https://docs.conda.io/en/latest/miniconda.html)
2, Download and build the required packages for your system
```
mkdir ~/MDAnalysis-wheel
cd MDAnalysis-wheel
conda create -n wheel-builder python=3.10 cython
conda activate wheel-builder
python -m pip wheel MDAnalysis==2.6.1 --cache-dir .
conda deactivate

```
3, Install the built .whl packages, into Blender’s bundled python. The path to your
4, Navigate to your Blender’s python folder 

`cd /Applications/Blender.app/Contents/Resources/3.6/python/bin/
`
5, Install the cached .whl into the bundled python that came with Blender.

`./python3.10 -m pip install MDAnalysis==2.6.1 --cache-dir ~/MDAnalysis-wheel
`

**Screenshots**
<img width="655" alt="Screenshot 2023-11-23 at 07 51 35" src="https://github.com/BradyAJohnston/MolecularNodes/assets/86617411/c90ad8db-023a-46e6-b8a0-2df1e7e4bda0">
<img width="253" alt="Screenshot 2023-11-23 at 07 51 50" src="https://github.com/BradyAJohnston/MolecularNodes/assets/86617411/b1f70866-7a00-4f19-838f-9c4a259b45b6">

**Desktop (please complete the following information):**
 - OS:  MacOS 14.1.1 (23B2082)
 - Hardware: Macbook M3 max,
 - Blender Version: Blender 4.0.1 and Blender 3.6.5 LTR
- MolecularNodes Version: 2.11.0 and 2.22.2

*** Errors in the console***

```
Python: Traceback (most recent call last):
  File "/Users/myang/Library/Application Support/Blender/3.6/scripts/addons/MolecularNodes/density.py", line 226, in execute
    vol = load(
  File "/Users/myang/Library/Application Support/Blender/3.6/scripts/addons/MolecularNodes/density.py", line 199, in load
    vdb_file = map_to_vdb(file, invert=invert, world_scale=world_scale)
  File "/Users/myang/Library/Application Support/Blender/3.6/scripts/addons/MolecularNodes/density.py", line 129, in map_to_vdb
    grid = map_to_grid(file, invert=invert)
  File "/Users/myang/Library/Application Support/Blender/3.6/scripts/addons/MolecularNodes/density.py", line 48, in map_to_grid
    volume = mrcfile.read(file)
  File "/Users/myang/.local/lib/python3.10/site-packages/mrcfile/load_functions.py", line 157, in read
    with open(name, mode='r', permissive=True) as mrc:
  File "/Users/myang/.local/lib/python3.10/site-packages/mrcfile/load_functions.py", line 139, in open
    return NewMrc(name, mode=mode, permissive=permissive,
  File "/Users/myang/.local/lib/python3.10/site-packages/mrcfile/mrcfile.py", line 109, in __init__
    self._open_file(name)
  File "/Users/myang/.local/lib/python3.10/site-packages/mrcfile/mrcfile.py", line 126, in _open_file
    self._iostream = open(name, self._mode + 'b')
FileNotFoundError: [Errno 2] No such file or directory: ''

```

I think the biotite and starfile is mising, MDanalysis seems working ok.

Could you please give me some suggestions about what I can do?
Thank you very much!

**Comment by BradyAJohnston** [comment]
Hi @MLeiY, thanks for the bug report and the error codes. I can't figure out what might be causing this, but I am working on the 4.0 compatible version which I should hopefully release this weekend, which should hopefully fix this issue.

It sounds like you might be able to in the meantime install 2.10.0 and install `biotite`, then upgrade the version if you need.

**Comment by BradyAJohnston** [comment]
See if the Blender 4.0 compatible version helps: https://github.com/BradyAJohnston/MolecularNodes/releases/tag/v4.0.0

**Comment by BradyAJohnston** [comment]
I was able to finally reproduce this error on another machine, and have released (yet another) version which should finally fix it (v4.0.1): https://github.com/BradyAJohnston/MolecularNodes/releases
